### ISSUE TITLE
Fails to display error message on Fail

### ISSUE THREAD

**Issue by arikrak** [teoljungberg/minitest-fail-fast#1]
When it reaches a failure, it just displays an interrupt message:

```
.................F
 ../../.rvm/gems/ruby-2.0.0-p451/gems/minitest-fail-fast-0.0.1/lib/minitest/fail_fast_plugin.rb:29:in `record': Interrupt (Interrupt)
```

I'll post a possible fix.

**Comment by teoljungberg** [comment]
Hm, It shouldn't. Which version of minitest are you on?

I couldn't reproduce the problem with this test, can you?

```
$ ruby -Ilib tmp/test.rb -f
Run options: -f --seed 33234

# Running:

.........F
Interrupted. Exiting...


Finished in 0.001388s, 7204.6110 runs/s, 7204.6110 assertions/s.

  1) Failure:
FooTest#test_50 [tmp/test.rb:8]:
I failed you

10 runs, 10 assertions, 1 failures, 0 errors, 0 skips
```

``` ruby
require "minitest/autorun"

class FooTest < Minitest::Test
  N = 100
  N.times do |i|
    define_method "test_#{i}" do
      if i == N / 2
        refute 1, "I failed you"
      else
        assert 1
      end
    end
  end
end
```

**Comment by arikrak** [comment]
Actually that worked. But when I ran the tests on a Rails 4.2.0 app (with Minitest 5.5.1), I got the previous message, along with an interrupt message. I was able to print out the error message by changing `io.puts` to `io.puts result.failures` on [line 28](https://github.com/teoljungberg/minitest-fail-fast/blob/master/lib/minitest/fail_fast_plugin.rb#L28).

**Comment by teoljungberg** [comment]
All right, I can reproduce this in a Rails 4.2 app aswell. 
Since it works in pure ruby the problem doesn’t lie in either the gem or minitest itself.
I think that the culprit might be in `minitest-rails` and if it changes how minitests test-runner works. But I have not idea.

**Comment by arikrak** [comment]
Hmm.. it seems I was able to fix the issue by updating Rails to 4.2.1 and Minitest to 5.6.1. So no need to change any code!

**Comment by teoljungberg** [comment]
Glad to hear that it worked out, thanks!
